chart/CHANGELOG.txt

```diff
@@ -0,0 +1,70 @@
+Airflow Helm Chart 1.1.0, 2021-07-25
+------------------------------------
+
+New Features
+""""""""""""
+- Chart: Allow using krb5.conf with ``CeleryExecutor`` (#16822)
```

Comment:
```suggestion
- Chart: Allow using ``krb5.conf`` with ``CeleryExecutor`` (#16822)
```

Reply:
Thanks @kaxil. For some reason I figured the commit messages should be left as-is. I'll keep this in mind for next time.